src/components/BusinessTable/BusinessTable.jsx

```diff
@@ -62,8 +71,46 @@ const BusinessTable = () => {
     'Created Date',
     'City',
   ];
-
   const tableHeaders = TABLE_HEADERS.map(tableHeader => <th key={tableHeader}>{tableHeader}</th>);
+  const [checkedItems, setCheckedItems] = useState(new Array(TABLE_HEADERS.length).fill(false));
+
+  const handleCheckboxChange = index => {
+    setCheckedItems(prev => {
+      const newCheckedItems = [...prev];
+      newCheckedItems[index] = !newCheckedItems[index];
+      return newCheckedItems;
+    });
+  };
+
+  const handleSelectAllChange = () => {
+    setCheckedItems(prev => {
+      const allChecked = prev.every(Boolean);
+      return prev.map(() => !allChecked);
+    });
+  };
+
+  const handleSendReminders = async () => {
+    const selectedBusinessIds = data.filter((_, index) => checkedItems[index]).map(item => item.id);
+
+    for (let i = 0; i < selectedBusinessIds.length; i++) {
+      try {
+        const requestData = {
+          business_id: selectedBusinessIds[i],
+          message: 'Message', 
+          timestamp: new Date().toISOString(),
+          been_dismissed: false,
+        };
+
+        await backend.post('/notification', requestData);
+
+        const response = await backend.get('/business');
```

Comment:
Not sure if we need the get request. Message us if you think there's a reason, otherwise you could get rid of it.